### PR TITLE
Use quoted form when including primitives/transaction.h and wallet/wallet.h

### PR DESCRIPTION
**Pull request by practicalswift**
Nit: We expect to find `primitives/transaction.h` locally in `src/`.

### CODE REVIEW

**Comment by jtimon** [comment]
trivial review ACK

**Comment by TheBlueMatt** [comment]
utACK

**Comment by fanquake** [comment]
utACK 0357ce0

**Comment by MeshCollider** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/10752/commits/0357ce0740e5fc168b408f8fd55aa7ba5531a082

**Comment by laanwj** [comment]
> Nit: We expect to find primitives/transaction.h locally in src/.

utACK as it matches the common convention better.

Though sometimes I wonder whether this convention is a good idea. `include <>` searches relative to the include paths, `include ""` relative to the directory of the current source file, then the include paths. We know `primitives/` is relative to the source root of the repository, so it doesn't really need to look anywhere else. Especially not if it is already in `primitives/`, e.g. 
- when `primitives/block.h` includes `"primitives/transaction.h"`, then it makes it look in `src/primitives/primitives`
- when `src/wallet/feebumper.h` includes `"primitives/transaction.h"`, it makes it look in `src/wallet/primitives`!

File: src/wallet/feebumper.h [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #ifndef BITCOIN_WALLET_FEEBUMPER_H
 #define BITCOIN_WALLET_FEEBUMPER_H
 
-#include <primitives/transaction.h>
+#include "primitives/transaction.h"
```
Comment (promag) [inline src/wallet/feebumper.h]: With quotes should be relative?
```cpp
#include "../primitives/transaction.h"
```

**Comment by jtimon** [comment]
The advantage of using #include "primitives/transaction.h" instead of simply #include "transaction.h" is that one day we may want to search and replace all  #include "primitives/transaction.h" say, to a new directory. If you moved primitives/block.o to the same directory at the same time it would be fine though.

**Comment by practicalswift** [comment]
Added a commit fixing the same thing when importing `wallet/wallet.h` :-)  Introduced in 97375727b8f3b7d26c7c813630a6139005b5c5c9.

**Comment by practicalswift** [comment]
Build errors seem to be unrelated?

**Comment by practicalswift** [comment]
Build error unrelated. May I request a re-build? :-)

**Comment by laanwj** [comment]
If you don't mind, I'm going to close this. There has been talk of moving to absolute inclusion (e.g. `#include <>`) instead on the longer run (https://github.com/bitcoin/bitcoin/pull/10976#issuecomment-322397113) and I think that'd be superior.

**Comment by practicalswift** [comment]
Makes sense! :-)

**Pull request closed without merging**